crates/holochain/Cargo.toml

```diff
@@ -110,6 +110,10 @@ tx5-signal-srv = { version = "=0.0.5-alpha", optional = true }
 bytes = { version = "1", optional = true }
 reqwest = { version = "0.11.2", features = ["json"], optional = true }
 
+# TODO: make optional?
+aitia = { path = "../aitia" }
+hc_sleuth = { path = "../hc_sleuth" }
```

Comment:
versions for path deps here?

Reply:
I think even if the release automation doesn't mind, these are wanted when the crate is published